app/views/questions/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class=" question ">
   <%= render partial: 'shared/error_messages', locals: { model: @question}%>
-  <%= render partial: 'users/show-mini', locals: {user: User.find(@question.user_id), extra_data: "Published: #{@question.published_at.strftime('%m/%d %I:%M %p')}"} %>
-  <div class="title"><%= @question.title %></div>
+  <%= render partial: 'users/show-mini', locals: {user: User.find(@question.user_id), extra_data: "Published: #{@question.published? ? @question.published_at.strftime('%m/%d %I:%M %p') : "nil"}"} %>
+  <div class="title"><%= @question.title %><%="(unpublished)" if @question.reports.size >= REPORTS_THRESHOLD %></div>
```

Comment:
@question.reports.size >= REPORTS_THRESHOLD
can be @question.reports.count 
Make a method, if required

Reply:
done